test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -435,6 +435,38 @@ private static void AssertElementsAreJoined(Element element, Element otherElemen
             Assert.AreEqual(expected, arejoined);
         }
 
+        [Test]
+        [TestModel(@".\Element\elementJoin.rvt")]
+        public void CanSuccessfullySwitchOrderOfTwoJoinedElements()
```

Comment:
not sure we need `Successfully` in every test name, would read easier without it